autoload/jsdoc.vim

```diff
@@ -50,6 +50,10 @@ endif
 if !exists('g:jsdoc_allow_shorthand')
   let g:jsdoc_allow_shorthand = 0
 endif
+" Use seperator between @param name and description.
+if !exists('g:jsdoc_param_description_seperator')
+	let g:jsdoc_param_description_seperator = " - "
```

Comment:
Could you change default behavior?
I don't like bc break.
`jsdoc_param_description_seperator` should be `""`